packages/calypso-e2e/src/lib/pages/signup/signup-pick-plan-page.ts

```diff
@@ -27,7 +27,7 @@ export class SignupPickPlanPage {
 	 * @param {Plans} name Name of the plan.
 	 * @returns {Promise<SiteDetails>} Details of the newly created site.
 	 */
-	async selectPlan( name: Plans ): Promise< SiteDetails > {
+	async selectPlan( name: Plans ): Promise< NewSiteResponse > {
```

Comment:
Unify the expected response - UserSignupPage already returns the response and this page should too.